website/src/pages/swiss-wastewater/flu.astro

```diff
@@ -37,23 +35,41 @@ const lapisUrl = `${wastewaterConfig.lapisBaseUrl}/influenza`;
                     <div class='m-4 flex flex-col gap-4'>
                         <>
                             <h2 class='text-2xl'>{reference}</h2>
-                            {(['nucleotide', 'amino acid'] satisfies SequenceType[]).map((sequenceType) => (
-                                <GsWastewaterMutationsOverTime
-                                    title={`${sequenceType} mutations`}
-                                    lapisFilter={{
-                                        reference,
-                                        versionStatus: 'LATEST_VERSION',
-                                        isRevocation: false,
-                                    }}
-                                    sequenceType={sequenceType}
-                                >
-                                    <InfluenzaWastewaterInfo
-                                        slot='infoText'
-                                        reference={reference}
-                                        sequenceType={sequenceType}
-                                    />
-                                </GsWastewaterMutationsOverTime>
-                            ))}
+                            <details>
+                                <summary class='cursor-pointer font-bold'> Nucleotide Mutations </summary>
+                                <p>
```

Comment:
Why do we have that `<p>` there? I guess that we don't need it? If we do need it, let's change it to a div (for better semantics)